src/constants/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = {
+  metadataColumns: {
```

Comment:
Could we add a comment here that points to this file on GitHub https://github.com/alphagov/pay-java-commons/blob/master/model/src/main/java/uk/gov/pay/commons/model/charge/ExternalMetadata.java and mentions that this is the frontend equivalent

It might also be useful to use the same naming so that it's clear they're talking about the same thing
```js
exteranlMetadata: {
  MAX_KEY_VALUE_PAIRS: x,
  MIN_KEY_LENGTH : x,
  MAX_KEY_LENGTH: x,
  MAX_VALUE_LENGTH: x
}
```

Self service can then take those values and map them to the specific language it's using it for (reporting columns).

Reply:
Good point - updated as per suppestion.